packages/flutter_test/test/utils/leak_tracking_utils.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+import 'package:leak_tracker/leak_tracker.dart';
+
+final List<LeakTrackedClass> notGcedStorage = <LeakTrackedClass>[];
+
+LeakTrackedClass notGCed = LeakTrackedClass();
+
+class StatelessLeakingWidget extends StatelessWidget {
+  StatelessLeakingWidget({super.key}) {
+    // ignore: unused_local_variable
```

Comment:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#comment-all-ignores